### PR TITLE
first version

### PR DESCRIPTION
**Pull request by jgfidelis**

### CODE REVIEW

File: node/globals.ts [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+  interface CustomIOContext extends IOContext {
+    segment?: SegmentData
+  }
+
+  interface Property {
+    name: string
+    values: [string]
+  }
+
+  // interface SkuSpecification {
```
Comment (firstdoit) [inline node/globals.ts]: :eyes:

File: node/globals.ts [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+  //   fieldName: Promise<TranslatableMessage>
+  //   fieldValues: [Promise<TranslatableMessage>]
+  // }
+
+  // interface ProductSpecification {
```
Comment (firstdoit) [inline node/globals.ts]: :eyes:

File: node/clients/checkout.ts [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+    this.post<OrderForm>(
+      this.routes.simulation(this.getChannelQueryString()),
+      simulation,
+      {
+        metric: 'checkout-simulation',
+      }
+    )
+
+  protected post = <T>(url: string, data?: any, config: RequestConfig = {}) => {
+    // config.headers = {
```
Comment (firstdoit) [inline node/clients/checkout.ts]: :eyes:

File: node/clients/checkout.ts [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+      headers: {
+        ...(options && options.headers),
+        // ...(ctx.storeUserAuthToken
+        //   ? { VtexIdclientAutCookie: ctx.storeUserAuthToken }
+        //   : null),
+      },
+    })
+  }
+
+  // private getCommonHeaders = () => {
```
Comment (firstdoit) [inline node/clients/checkout.ts]: 👀

File: graphql/types/Product.graphql [hunk 5]
```diff
@@ -0,0 +1,315 @@
...
+  unitMultiplier: Float
+  kitItems: [KitItem]
+  images(quantity: Int = 10): [Image]
+  videos: [Video]
+  sellers: [Seller]
+  variations: [Property]
+  attachments: [Attachment]
+  # productName: String @translatable
+  # productDescription: String @translatable
+  # brandName: String @translatable
```
Comment (klzns) [inline graphql/types/Product.graphql]: Why is that commented out?

File: node/clients/checkout.ts [hunk 6]
```diff
@@ -0,0 +1,61 @@
...
+} from '@vtex/api'
+import { statusToError } from '../utils'
+
+export class Checkout extends JanusClient {
+  public constructor(ctx: IOContext, options?: InstanceOptions) {
+    super(ctx, {
+      ...options,
+      headers: {
+        ...(options && options.headers),
+        // ...(ctx.storeUserAuthToken
```
Comment (firstdoit) [inline node/clients/checkout.ts]: :eyes:

File: node/resolvers/paths.ts [hunk 7]
```diff
@@ -0,0 +1,77 @@
+import { join } from 'ramda'
+
+const paths = {
+  /** Checkout API
+   * Docs: https://documenter.getpostman.com/view/18468/vtex-checkout-api/6Z2QYJM
+   */
+  addItem: (account: any, { orderFormId }: any) => `${paths.orderForm(account)}/${orderFormId}/items`,
+  changeToAnonymousUser: (account: any, { orderFormId }: any) => `http://${account}.vtexcommercestable.com.br/checkout/changeToAnonymousUser/${orderFormId}`,
```
Comment (klzns) [inline node/resolvers/paths.ts]: urgh, i hate this file

File: node/tsconfig.json [hunk 8]
```diff
@@ -0,0 +1,42 @@
...
+    "esModuleInterop": true,
+    "lib": [
+      "es2017",
+      "esnext.asynciterable"
+    ],
+    "listEmittedFiles": true,
+    "module": "commonjs",
+    "moduleResolution": "node",
+    "noImplicitAny": true,
+    "noImplicitReturns": true,
```
Comment (klzns) [inline node/tsconfig.json]: Can change all that to:

```json
{
  "extends": "@vtex/tsconfig",
  "include": [
    "./typings/*.d.ts",
    "./**/*.tsx",
    "./**/*.ts"
  ]
}
```

`yarn add @vtex/tsconfig -D`

File: node/resolvers/catalog/sku.ts [hunk 9]
```diff
@@ -0,0 +1,114 @@
...
+        Videos
+      ),
+
+    nameComplete: (
+      { nameComplete, itemId }: CatalogItem,
+      _: any,
+      { clients: { segment } }: Context
+    ) => toSKUIOMessage('nameComplete')(segment, nameComplete, itemId),
+
+    // skuSpecifications: (
```
Comment (firstdoit) [inline node/resolvers/catalog/sku.ts]: 👀

File: graphql/types/Product.graphql (hunk 5 above)
Comment (jgfidelis) [reply graphql/types/Product.graphql]: this was removed

File: node/service.json [hunk 10]
```diff
@@ -0,0 +1,8 @@
+{
+  "stack": "nodejs",
+  "memory": 1536,
+  "ttl": 30,
+  "timeout": 20,
```
Comment (firstdoit) [inline node/service.json]: Tighterrrr 🏎

File: node/service.json [hunk 11]
```diff
@@ -0,0 +1,8 @@
...
+  "minReplicas": 30,
```
Comment (firstdoit) [inline node/service.json]: Let's try starting with 20?

File: policies.json [hunk 12]
```diff
@@ -0,0 +1,15 @@
+[
+  {
+    "name": "resolve-graphql",
+    "description": "Allows access to resolve a graphql request",
+    "statements": [
+      {
+        "actions": ["post", "get"],
+        "effect": "allow",
+        "resources": [
+          "vrn:vtex.store-graphql:{{region}}:{{account}}:{{workspace}}:/_v/graphql"
```
Comment (firstdoit) [inline policies.json]: Oops!

**Pull request merged**